egs/wsj/s5/steps/scoring/score_kaldi_wer.sh

```diff
@@ -18,7 +18,7 @@ max_lmwt=17
 iter=final
 #end configuration section.
 
-echo "$0 $@"  # Print the command line for logging
+echo "EDITED $0 $@"  # Print the command line for logging
```

Comment:
this may be not for commi

Reply:
I think I should put all the ` sed 's/ //g' | sed 's/|/ /g' ` in the `local/wer_output_filter`. So I'll revert every changes in the score files.